cni/network/network_windows.go

```diff
@@ -44,10 +44,15 @@ func handleConsecutiveAdd(args *cniSkel.CmdArgs, endpointId string, nwInfo *netw
 		log.Printf("[net] Found existing endpoint through hcsshim: %+v", hnsEndpoint)
 		log.Printf("[net] Attaching ep %v to container %v", hnsEndpoint.Id, args.ContainerID)
 
-		err := hcsshim.HotAttachEndpoint(args.ContainerID, hnsEndpoint.Id)
-		if err != nil {
-			log.Printf("[cni-net] Failed to hot attach shared endpoint[%v] to container [%v], err:%v.", hnsEndpoint.Id, args.ContainerID, err)
-			return nil, err
+		endpoint, _ := hcsshim.GetHNSEndpointByID(hnsEndpoint.Id)
```

Comment:
check if endpoint is nil

Reply:
Need to sync up with hns team to see whether its necessary to check hnsEndpoint.Id nil when hnsEndpoint is not nill. If it's necessary, let me add in another separate CR.